src/main/resources/tunnelmc.mixins.json

```diff
@@ -4,18 +4,21 @@
   "package": "me.THEREALWWEFAN231.tunnelmc.mixins",
   "compatibilityLevel": "JAVA_8",
   "client": [
-  	"interfaces.IMixinDimensionType",
-  	"interfaces.IMixinPlayerListS2CPacket",
-  	"interfaces.IMixinPlayerInteractEntityC2SPacket",
-  	"interfaces.IMixinEntityPositionS2CPacket",
-  	"interfaces.IMixinEntitySetHeadYawS2CPacket",
-  	"interfaces.IMixinCloseHandledScreenC2SPacket",
-  	"MixinMinecraftClient",
-  	"MixinMultiplayerScreen",
-  	"MixinClientConnection",
-  	"MixinKeyboard",
-  	"MixinAbstractClientPlayerEntity",
-  	"MixinGameRenderer",
-  	"MixinClientPlayerEntity"
+    "MixinAbstractClientPlayerEntity",
+    "MixinClientConnection",
+    "MixinClientPlayerEntity",
+    "MixinGameRenderer",
+    "MixinKeyboard",
+    "MixinMinecraftClient",
+    "MixinMultiplayerScreen",
+    "interfaces.IMixinCloseHandledScreenC2SPacket",
+    "interfaces.IMixinDimensionType",
+    "interfaces.IMixinEntityPositionS2CPacket",
+    "interfaces.IMixinEntitySetHeadYawS2CPacket",
+    "interfaces.IMixinPlayerInteractEntityC2SPacket",
+    "interfaces.IMixinPlayerListS2CPacket"
+  ],
+  "mixins": [
+    "MixinClientWorld"
```

Comment:
This isn't a personal preference; this is what my IDE defaults to when adding a Mixin through a shortcut. Can be changed.